spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkBatchScan.java

```diff
@@ -61,14 +56,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-abstract class SparkBatchScan implements Scan, Batch, SupportsReportStatistics {
+abstract class SparkBatchScan implements Scan, SupportsReportStatistics {
```

Comment:
Does it make sense to rename as `SparkScan`?

Reply:
Yeah, I think so. Changed.